src/docs/testing/index.md

```diff
@@ -203,6 +204,7 @@ To use `flutter_driver`, you must add the following block to your `pubspec.yaml`
 
 ```yaml
 dev_dependencies:
+  test:
```

Comment:
This addition isn't needed AFAIK

Reply:
Without this `import 'package:test/test.dart';` within `Writing integration tests` fails for me.

Reply:
Right you are! (There is an explicit import of the test package in the integration test file :)

For this entry, also change it to be:

```yaml
dev_dependencies:
  flutter_driver:
    sdk: flutter
  flutter_test:
    sdk: flutter
  test: ^1.5.1
```

Reply:
Done!